<?xml version="1.0"?>
<robot name="afa">

  <link name="bintel">
    <visual>
      <origin rpy="0.0 0 1.5707" xyz="0 0 0.008"/>
      <geometry>
        <mesh filename="package://bintel_ros/meshes/afa.stl"/>
      </geometry>
    </visual>
  </link>
  
</robot>
